src/compile.ts

```diff
@@ -50,25 +50,40 @@ const writeFile = (schema: Record<string, unknown>, name: string) => {
   fs.writeFileSync(target, JSON.stringify(schema, null, 2));
 };
 
+/**
+ * Returns a string with a markdown link to a schema's documentation.
+ */
+const prepareLink = (pathname: string) => {
+  const urlPath = pathname.replace('.yml', '');
+  const schemaName = startCase(last(urlPath.split('/')));
+  const link = `\n\nSee more: [${schemaName} Schema](https://schema.laboperator.com/schemas/${urlPath}) `;
+
+  return link;
+};
+
 /**
  * On hover documentation is based on the standard `description` field. In order
  * to provide rich text formatting within VS Code we can add a custom field.
  *
  * https://code.visualstudio.com/docs/languages/json#_use-rich-formatting-in-hovers
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const addMarkdownDescription = (obj: Record<string, any>) => {
+const addMarkdownDescription = (pathname: string, obj: any) => {
+  // The `obj` param can be an array for nested schemas, e.g. `allOf` field,
+  // but we for..in iterate over it anyway until it breaks 🙃
   for (const key in obj) {
     if (key === 'description' && typeof obj[key] === 'string') {
```

Comment:
why are we doing this check `typeof obj[key] === 'string'`?

Reply:
should we remove it?

Reply:
Ah, sorry, I failed to realize which function this is. We need this condition, because otherwise TS complains about invoking `replace`

Reply:
I guess I missed this since IntelliSense didn't show TS complaint. It happens when you run `build`